src/index.js

```diff
@@ -24,4 +25,14 @@ Object.defineProperty(module.exports, 'logger', {
 });
 
 export default ParseServer;
-export { S3Adapter, GCSAdapter, FileSystemAdapter, InMemoryCacheAdapter, NullCacheAdapter, RedisCacheAdapter, TestUtils, PushWorker, _ParseServer as ParseServer };
+export {
+  S3Adapter,
+  GCSAdapter,
+  FileSystemAdapter,
+  InMemoryCacheAdapter,
+  NullCacheAdapter,
+  RedisCacheAdapter,
+  TestUtils,
+  PushWorker,
+  _ParseServer as ParseServer
```

Comment:
i think you meant to add LRUCacheAdapter to this export?

Reply:
doh, yes

Reply:
I know, just trying to make sure I'm paying attention ;)